pkg/operation/botanist/component/clusterautoscaler/logging.go

```diff
@@ -15,34 +15,63 @@
 package clusterautoscaler
 
 import (
+	fluentresources "github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/filter"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/parser"
```

Comment:
Use `fluentbitv1alpha2parser` as import alias and enforce it in https://github.com/gardener/gardener/blob/master/.golangci.yaml#L49